AGENTS.md

```
@@ -18,12 +18,16 @@
 - Enable hooks: `pre-commit install` (clang-format + detect-secrets + docs link/TOC checks)
 - Docs linter (all docs): `python3 tools/docs/check_docs.py --mode link` and `--mode toc`
 - Pre-commit (changed files): `pre-commit run --all-files docs-link-check` and `docs-toc-check`
+ - Markdown linter: `make md-lint` (check) and `make md-fix` (autofix) — uses `.markdownlint.jsonc`
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)